src/equiv.js

```diff
@@ -1,219 +1,214 @@
 // Test for equality any JavaScript type.
 // Author: Philippe Rathé <prathe@gmail.com>
-QUnit.equiv = (function() {
-
-	// Call the o related callback with the given arguments.
```

Comment:
this file got a huge diff because the indentation level reduced here. To ignore the whitespace change, see this diff including the [`?w=1`](https://github.com/jquery/qunit/pull/735/files?w=1) parameter.